Source/WebKit/Shared/cf/CFTypes.serialization.in

```diff
@@ -69,7 +69,7 @@ additional_forward_declaration: typedef struct CF_BRIDGED_TYPE(id) __SecKeychain
 #endif
 
 additional_forward_declaration: typedef struct CF_BRIDGED_TYPE(id) __CVBuffer *CVPixelBufferRef
-[WebKitPlatform, CustomHeader,  AdditionalEncoder=StreamConnectionEncoder, ToCFMethod=result->toCF()] CVPixelBufferRef wrapped by WebKit::CoreIPCCVPixelBufferRef {
+[WebKitPlatform, CustomHeader, ToCFMethod=result->toCF()] CVPixelBufferRef wrapped by WebKit::CoreIPCCVPixelBufferRef {
```

Comment:
This attribute was previously misspelled (extra space before) and therefore the StreamEncoder wasn't ever being generated.